ui/keyboardMove/tests/keyboardSubmit.test.ts

```diff
@@ -631,8 +644,9 @@ describe('keyboardSubmit', () => {
   });
 
   test('with incorrect entry marks it wrong', () => {
-    input.classList.toggle = jest.fn() as any;
-    site.sound = { play: jest.fn() } as any;
+    vi.stubGlobal('site', { sound: { play: vi.fn() } });
```

Comment:
Much cleaner globals solution compared to the jest package.json globals business!